Sources/CustomerCenter/CustomerCenterConfigData.swift

```diff
@@ -73,6 +73,10 @@ public struct CustomerCenterConfigData {
             case defaultBody = "default_body"
             case defaultSubject = "default_subject"
             case dismiss = "dismiss"
+            case updateWarningTitle = "update_warning_title"
+            case updateWarningDescription = "update_warning_description"
+            case updateWarningUpdate = "update_warning_update"
+            case updateWarningIgnore = "update_warning_ignore"
```

Comment:
Not sure if we want to make these strings specific to the app update warning screen?